docs/shareseq/gex_rnaseq.rst

```diff
@@ -0,0 +1,250 @@
+
+	.. list-table::
+		:widths: 5 30 30 20
+		:header-rows: 1
+
+		* - Name
+		  - Description
+		  - Example
+		  - Default
+		* - **input_csv_file**
+		  - Sample Sheet (contains Sample, Lane, Index, Flowcell, [optional for mkfastq: Reference, Type])
+		  - "gs://fc-e0000000-0000-0000-0000-000000000000/sample_sheet.csv"
+		  -
+		* - **output_directory**
+		  - Output directory
+		  - "gs://fc-e0000000-0000-0000-0000-000000000000/shareseq_output"
+		  - Results are written under directory *output_directory* and will overwrite any existing files at this location.
+		* - run_mkfastq
+		  - If you want to run ``shareseq mkfastq``
+		  - true
+		  - true
+		* - run_count
+		  - If you want to run ``Chromap and STARsolo``
+		  - true
+		  - true
+		* - delete_input_bcl_directory
+		  - If delete BCL directories after demux. If false, you should delete this folder yourself so as to not incur storage charges
+		  - false
+		  - false
+		* - dual_index_type
+		  - Dual-index Paired-end flowcell workflow type
+		  - "auto" or "forward" or "reverse"
+		  - auto
+		* - barcode_mismatches
+		  - Number of allowed mismatches per index
+		  - 0,1 or 2
+		  - 1
+		* - use_bases_mask
+		  - Override the read lengths as specified in RunInfo.xml
+		  -
+		  -
+		* - read1_fastq_pattern
+		  - read1 fastq pattern
+		  - "_S*_L*_R1_001.fastq.gz"
+		  - "_S*_L*_R1_001.fastq.gz"
+		* - read2_fastq_pattern
+		  - read2 fastq pattern
+		  - "_S*_L*_R3_001.fastq.gz"
+		  - "_S*_L*_R3_001.fastq.gz"
+		* - index_fastq_pattern
+		  - index fastq pattern
+		  - "_S*_L*_R2_001.fastq.gz"
+		  - "_S*_L*_R2_001.fastq.gz"
+		* - docker_registry
+		  - Docker registry for shareseq_workflow. Options:
+
+                        - "quay.io/cumulus" for images on Red Hat registry;
+
+                        - "cumulusprod" for backup images on Docker Hub.
+
+		  - quay.io/cumulus
+		  - quay.io/cumulus
+		* - mkfastq_docker_registry
+		  - docker registry for specifically mkfastq.
+		  - gcr.io/broad-cumulus
+		  - gcr.io/broad-cumulus
+		* - zones
+                  - Google cloud zones
+                  - "us-central1-a us-west1-a"
+                  - "us-central1-a us-central1-b us-central1-c us-central1-f us-east1-b us-east1-c us-east1-d us-west1-a us-west1-b us-west1-c"
+		* - backend
+                  - Cloud backend for file transfer. Available options:
+
+                    - "gcp" for Google Cloud;
+                    - "aws" for Amazon AWS;
+                    - "local" for local machine.
+                  - "gcp"
+                  - "gcp"
+		* - acronym_file
+		  - | The link/path of an index file in TSV format for fetching preset genome references, chemistry whitelists, etc. by their names.
+		    | Set an GS URI if *backend* is ``gcp``; an S3 URI for ``aws`` backend; an absolute file path for ``local`` backend.
+		  - "s3://xxxx/index.tsv"
+		  - "gs://gred-cumulus-output/ref-data/shareseq/index.tsv"
```

Comment:
Can you replace 'gred-cumulus-output' to 'xxxx' for now? We will replace this line later.